### ISSUE TITLE
Regression: -s NO_FILESYSTEM=1 can't be used with Google Closure on 1.38.31

### ISSUE THREAD

**Issue by ebraminio** [emscripten-core/emscripten#8677]
Steps to reproduce:
1. Have Manjaro or Arch that has emscripten 1.38.31 on its package manager, and install Google Closure from aur or somewhere
2. `git clone https://github.com/harfbuzz/harfbuzzjs && cd harfbuzzjs && ./build.sh`
  (very lightweight script, you can inspect CMakeLists.txt and build.sh easily, harfbuzz itself is a popular package you are already using in your Linux or browser)

Actual:
```
error: undefined symbol: $FS
warning: To disable errors for undefined symbols use `-s ERROR_ON_UNDEFINED_SYMBOLS=0`
```

Still broken if I add that also but fixes if I remove `-s NO_FILESYSTEM=1`, remove the /build folder after each change however

Expected:
If I downgrade to 1.38.30, it works again.

**Comment by kripken** [comment]
We fixed a bunch of issues here - please test if this is still a problem on 1.38.33 (you need to install it from the new builders, see https://groups.google.com/d/msg/emscripten-discuss/WhDtqVyW_Ak/8DfDnfk0BgAJ )

**Comment by ebraminio** [comment]
Should I get that using 
```
cd emscripten/ 
git fetch 
git checkout origin/incoming 

cd ../emscripten-fastcomp/ 
git fetch 
git checkout origin/incoming 
cd tools/clang 
git fetch 
git checkout origin/incoming 
cd ../../build 
make -j4 
```
?

Is there any docker image with the version? Just was wondering providing some docker with the latest emscripten can be very cool and making the testing whole lot easier, the image themselves can uploaded using a bot using quay.io so no manual interference each time even.

**Comment by ebraminio** [comment]
Fixed in 1.38.32, yay! Thanks
